modules/copying-files-pods-and-containers.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
```

Comment:
@sferich888 it was discussed initially but not on the radar now.

Reply:
@vikram-redhat, and for my next trick...

Reply:
Based on this conversation, I will leave this in place for now and we can also add this module to the CLI man pages documentation later. I think that it would be useful in both locations.